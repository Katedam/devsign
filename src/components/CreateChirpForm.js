import React from 'react';
import PropTypes from 'prop-types';
import { Label } from '../styles/mainStyle';
import { ChirpForm, NewChirpButton, NewChirpInput } from '../styles/createChirpForm';

function CreateChirpForm({ text }) {
  return (
    <section>
      <ChirpForm>
        <Label htmlFor="newChirpText">Name</Label>
        <NewChirpInput readOnly={true} type="text" name="newChirpText" id="newChirpText" value={text} />
        <Label htmlFor="newChirp"></Label>
        <NewChirpButton id="newChirp">Chirp</NewChirpButton>
      </ChirpForm>
    </section>
  );
}

CreateChirpForm.propTypes = {
  text: PropTypes.string
};

export default CreateChirpForm;
