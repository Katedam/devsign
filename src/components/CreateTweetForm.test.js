import React from 'react';
import CreateChirpForm from './CreateChirpForm';
import renderer from 'react-test-renderer';

describe('CreateChirpForm component', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <CreateChirpForm />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
