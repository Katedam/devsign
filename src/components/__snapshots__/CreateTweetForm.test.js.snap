// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateTweetForm component matches a snapshot 1`] = `
<section>
  <form
    className="sc-bxivhb dBAuwS"
  >
    <input
      className="sc-EHOje iUmBVY"
      name="text"
      readOnly={true}
      type="text"
    />
    <button
      className="sc-bZQynM duCUaL"
    >
      Chirp
    </button>
  </form>
</section>
`;
